helm-chart/ray-cluster/templates/raycluster-cluster.yaml

```diff
@@ -123,8 +123,12 @@ spec:
         initContainers: {{- toYaml $values.initContainers | nindent 10 }}
         {{- end }}
         containers:
-          - volumeMounts: {{- toYaml $values.volumeMounts | nindent 12 }}
-            name: ray-worker
+          - name: ray-worker
+            {{ if $values.volumeMounts }}
+            volumeMounts: {{ toYaml (uniq (concat $values.volumeMounts $.Values.worker.volumeMounts)) | nindent 12 }}
```

Comment:
Why do we need to concatenate the `volumeMounts` from `$values.volumeMounts` and  `$.Values.worker.volumeMounts`? Maybe we can just make it optional. Something like:

```
{{ if $values.volumeMounts }}
volumeMounts : {{- toYaml $values.volumeMounts | nindent 12 }}
{{- end }}
```

Reply:
Because from what I can tell for proper Ray logs to exist we need the volume and volumeMounts for it. If we don't, then I'm happy to remove them but then I'm concerned where the logs will go exactly?

Reply:
The volume is optional but recommended to have. Most users do not use volumes to persist logs in production. If users want to persist logs, they should use K8s-native logging tools, such as FluentD, instead.

Reply:
Ahh, cool, I'll remove the merge then!

Reply:
@kevin85421, merge has been removed and now it just checks if it exists.